src/components/Calendar/assign-computers/computers-list.js

```diff
@@ -1,46 +1,46 @@
-import React,{useEffect, useState} from 'react'
-import "./computers-list-style.css" 
-import {students} from '../data'
-import {mentors} from '../data'
- 
-const ComputersList = (props)=>{
+import React, { useEffect, useState } from 'react'
+import '../../../less/computers-list-style.less'
```

Comment:
Changing the style import from the CSS file to LESS file is the only actual change in this file; everything else is just automagic whitespace changes by the linter.